js/js-flipper/src/client.ts

```diff
@@ -190,6 +190,11 @@ export class FlipperClient {
   }
 
   stop() {
+    if (this.reconnectionTimer) {
+      clearTimeout(this.reconnectionTimer);
+      this.reconnectionTimer = undefined;
+    }
+
     if (!this.ws) {
```

Comment:
Note: Important to keep the clearTimeout above this condition, since the `reconnect` method sets the `this.ws = undefined`.